data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +432,312 @@ mission "Giaru Gegno: Revisit"
+			choice
+				`	"Dueyu human..?"`
+				`	"I am human."`
+				`	(Say nothing.)`
+					goto "silence"
+			`	The Scin looks at you in what you guess to be fascination. They grab the device and draw yet another picture. After setting it down, you recognize the familiar image of a Gegno Vi warrior.`
+			choice
+				`	"Gegno Vi?"`
+				`	"Dueyu Vi?"`
+			action
+				set "scin: imagery"
+			`	Looking at you a bit confused, they draw another portrait that looks like the shorter leather-wearing Gegno from their homeworld and move that image together with the images of the Vi and the Scin. "Gegno," they say, and begin to think to themselves. A brief moment passes, and they begin drawing on the device again. After turning it back to you, you see both a green-colored patch and a brown-colored patch. "Vie," the Scin enunciates, pointing to the brown. "Sinn," they say, now pointing to the green. Afterwards, they point to the group of the three Gegno portraits. "Gegno."`
+			`	Once they mark down a few more notes, they lower their head, and the other Scin usher you back to the elevator, activating it to lower you back to the surface. You're escorted back to your ship, and along the way are a few Scin you haven't seen yet. Their armor is much different than that of the others, characterized by darker shades, sharper edges, and several gadgets, but what is most notable is their possession of sickle-like blade weapons.`
+			`	Once you return to your ship, the Scin return to theirs, but remain on the planet. Although just as strange, this encounter went far better than the one with the Vi.`
+				decline
+
+			label "pistol"
+			action
+				set "scin: provocative"
+			`	The situation you find yourself in is rather precarious - inside an alien species' vessel, strange devices - anything could happen next. You decide to reach for your pistol, but as soon as you do, the first thing you notice is a curved blade around your neck. Slowly turning your head, your eyes are met with the cold, expressionless look of yet another Scin's face, whom you haven't seen yet. They continue to hold the blade to your throat as they activate the elevator again, lowering you back to the surface. Their armor is considerably different from the rest, characterized by darker shades, sharper edges, and several gadgets.`
+			`	They escort you back to your ship, and along the way, several more Scin stand guard, wearing the same armor. Once you are at your ship, the blade is moved away from your neck. Getting a better look at it, the blade is shaped like a sickle, as opposed to a sword, meaning there was little chance you could have escaped being decapitated if they chose to use it. Boarding your ship in a hurry, you prepare to take off, and you notice the two Scin warships also begin to power up. It's clear you've overstayed your welcome.`
```

Comment:
```suggestion
			`	They escort you back to your ship, and, along the way, several more Scin stand guard in the same armor. Once you are at your ship, the blade is moved away from your neck. Getting a better look at it, the blade is shaped like a sickle, as opposed to a sword, meaning there was little chance you could have escaped being decapitated if they had chosen to use it. Boarding your ship in a hurry, you prepare to take off, and you notice the two Scin warships also begin to power up. It's clear you've overstayed your welcome.`
```

Reply:
I don't think that first comma addition is correct. The rest is an improvement on the previous wording.